nixos/modules/config/sysctl.nix

```diff
@@ -21,11 +21,19 @@ in
   options = {
 
     boot.kernel.sysctl = mkOption {
+      type = types.submodule {
+        freeformType = types.attrsOf sysctlOption;
+        options."net.core.rmem_max" = mkOption {
+          type = types.ints.unsigned // {
+            merge = loc: defs: foldl (a: b: lib.max a.value b.value) {value = 0;} defs;
```

Comment:
Since this merge function does not consider priority (usually using [`filterOverrides`][1]), users cannot use `mkForce` to set this value.

If priority is considered, IMHO these four services defining `` can ramain unchanged.


[1]: https://github.com/NixOS/nixpkgs/blob/f6072a8d7b1520194453e27be1fd59a28c751a7f/lib/modules.nix#L832